user_guides/jakartaee/src/main/jbake/content/rules.adoc

```diff
@@ -686,6 +686,6 @@ required for Jakarta EE 9:
 * jakarta.ws.rs.core
 * jakarta.ws.rs.ext
 * jakarta.ws.rs.sse
-* jakarta.xml.registry
-* jakarta.xml.registry.infomodel
+* javax.xml.registry
```

Comment:
We should remove the (pruned for EE 9) javax.xml.registry class references.

Reply:
I believe this PR can be merged after this is resolved to remove javax.xml.registry & javax.xml.registry.infomodel